src/js/ia/IAPage.js

```diff
@@ -360,6 +361,10 @@
                         }
                     });
 
+                    $("body").on('click', ".userpage-link a", function(evt) {
```

Comment:
Why `$("body").on(...)` for click handlers? Can't you bind directly to the links?
```
$(".userpage-link a").on('click', function(){})
```